.github/workflows/unit-tests.yaml

```diff
@@ -73,7 +74,7 @@ jobs:
         paths: 'junit-reports/report.xml'
 
     - name: Report test failures to Jira
-      if: (!cancelled())
+      if: (!cancelled()) && matrix.arch != 'arm64' # junit2jira needs arm release
```

Comment:
Can you please file a ticket and mention it as TODO here?
cc @janisz

Reply:
done